README.md

```
@@ -84,15 +130,18 @@ This is intelligence no IDE provides in one shot, enabling AI to debug more effe
 ```bash
 # Stop when specific condition occurs, with full trace to that point
 ./vendor/bin/xdebug-debug --break=User.php:85:$id==0 -- php register.php
 
-# Multiple conditions
+# Multiple conditions (comma-separated)
 ./vendor/bin/xdebug-debug --break=Auth.php:42:$token==null,User.php:85:$id==0 -- php app.php
 
-# With AI analysis
-./vendor/bin/xdebug-debug --break=Cart.php:89:$total<0 --claude -- php app.php
+# Auto-exit with trace file output
+./vendor/bin/xdebug-debug --break=Cart.php:89:$total<0 --exit-on-break -- php app.php
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)